### ISSUE TITLE
Fact-Store json if statement checks could use enums rather than hardcoded strings 

### ISSUE THREAD

**Issue by zmhassan** [AICoE/log-anomaly-detector#99]
**Is your feature request related to a problem? Please describe.**
Provide concrete types and avoid checking for 'false' from json response coming from the fact-store

```python
print("data['false_anomaly'] is {} ".format(data['false_anomaly']))
        # TODO: You should throw an exception
        #  if you get a value that other then 'false' or 'true'
        if data['false_anomaly'] == 'false':
```

**Describe the solution you'd like**
Provide concrete types and avoid checking for 'false' from json response coming from the fact-store

**Describe alternatives you've considered**
N/A

**Additional context**
This was a result of getting feedback from PR #75  which @vpavlin advised that we should check for False value using bool or concrete types.

**Comment by zmhassan** [comment]
This is being addressed in PR #75 So I'll close this once thats merged.

**Comment by zmhassan** [comment]
Closing issue as its complete
